internal/models/objectmeta/statuscondition/object_condition_status.go

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Rename package `statuscondition` to `status` and the import alias for it should be `statusmodel`.

Also consider merging the contents of the 3 files under this package into one single file since these all are used only for conditions. You may use the file name `conditions`

We can omit the `object` from the name.